docs/conf.py

```diff
@@ -287,6 +287,7 @@
     'tox': ('https://tox.readthedocs.io/en/latest', None),
     'twisted': ('https://twistedmatrix.com/documents/current', None),
     'twistedapi': ('https://twistedmatrix.com/documents/current/api', None),
+    'cssselect': ('https://cssselect.readthedocs.io/en/latest', None),
```

Comment:
:lipstick: A-Z

Reply:
............................................ah! Well that took me embarrassingly long to parse.

Reply:
............................................ah! Well that took me embarrassingly long to parse.